webapp/src/components/workspaceSwitcher/workspaceOptions.scss

```diff
@@ -0,0 +1,67 @@
+.workspaceOption {
+    padding: 0px 20px;
+    display: flex;
+    flex-direction: column;
+    cursor: pointer;
+
+    &:first-child {
+        margin-top: 8px;
+    }
+
+    &:last-child {
+        margin-bottom: 8px;
+    }
+
+    &.focused {
+        background-color: rgba(var(--button-bg-rgb), 0.08);
+    }
+
+    .workspaceTitle {
+        color: rgb(var(--center-channel-color-rgb));
+        font-size: 14px;
+    }
+
+    .boardCount {
+        color: rgba(var(--center-channel-color-rgb), 0.9);
+        font-size: 10px;
+        text-transform: uppercase;
+    }
+}
+
+.WorkspaceOptions__indicators {
```

Comment:
I would expect all this to be something like:
```css
.WorkspaceOptions {
    &__indicators{
        ...
    }
    &__control {
        ...
    }
    ...
}
```

Reply:
Oh, nice! Didn't knew we could do this.